generator/graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/internal/types/generateDirective.kt

```diff
@@ -66,43 +69,51 @@ internal fun generateEnumValueDirectives(generator: SchemaGenerator, field: Fiel
 
 private fun getDirective(generator: SchemaGenerator, directiveInfo: DirectiveMetaInformation): GraphQLAppliedDirective {
     val directiveName = directiveInfo.effectiveName
-    val directiveFromHook = generator.config.hooks.willGenerateDirective(directiveInfo)
-        ?.let { generator.directives.putIfAbsent(directiveName, it) }
-
-    val directive = directiveFromHook ?: generator.directives.computeIfAbsent(directiveName) {
-        val builder = GraphQLDirective.newDirective()
-            .name(directiveInfo.effectiveName)
-            .description(directiveInfo.directiveAnnotation.description)
-            .repeatable(directiveInfo.repeatable)
+    val directive = generator.directives.computeIfAbsent(directiveName) {
```

Comment:
previously we were always executing the `willGenerateDirective` hook instead of actually reading the previously generated value

Reply:
fixed in https://github.com/ExpediaGroup/graphql-kotlin/pull/1818